model_server/model_server.ipynb

```diff
@@ -60,13 +69,12 @@
    "outputs": [],
    "source": [
     "%%nuclio cmd -c\n",
-    "python -m pip install -U kfserving \n",
-    "python -m pip install numpy cloudpickle v3io"
+    "python -m pip install kfserving"
    ]
```

Comment:
@yjb-ds we sould not use kfserving, rather stick to the new base model server in MLRun

also you seem to work on an old version where the v3io was external, its now part of the model server class see in `mlrun/examples/xgb_server`